README.md

```diff
@@ -25,11 +26,16 @@ const producer = Producer.create({
 });
 
 // create custom producer (supporting all opts as per the API docs: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SQS.html#constructor-property)
+AWS.config.update({
+  accessKeyId: config.queue.aws_access_key_id,
+  secretAccessKey: config.queue.aws_secret_access_key,
```

Comment:
Presumably, `config` is from your own code?

We probably want to match the original README here:

```
accessKeyId: 'yourAccessKey',
secretAccessKey: 'yourSecret'
```

Reply:
oops, you are right. I will change it. 
thx :)